os/sdk-modifying-coreos.md

```diff
@@ -31,17 +31,15 @@ git config --global user.name "Your Name"
 
 The `cork` utility, included in the CoreOS [mantle](https://github.com/coreos/mantle) project, is used to create and work with an SDK chroot.
 
-In order to use this utility, you must additionally have the [golang](https://golang.org/) toolchain installed and configured correctly.
-
 First, install the cork utility:
 
 ```sh
-git clone https://github.com/coreos/mantle
-cd mantle
-./build cork
-mkdir ~/bin
-mv ./bin/cork ~/bin
-export PATH=$PATH:$HOME/bin
+curl https://github.com/coreos/mantle/releases/download/v0.7.0/cork-0.7.0-amd64
+curl https://github.com/coreos/mantle/releases/download/v0.7.0/cork-0.7.0-amd64.sig
+gpg --verify cork-0.7.0-amd64.sig cork-0.7.0-amd64
```

Comment:
We can't assume the user will already have imported the signing key, and we should be explicit about what output to expect.